doc/dev/mgmt/tests.md

```diff
@@ -10,42 +10,40 @@
     -   [Running tests in live mode](#running-tests-in-live-mode)
 - [Using the Azure Python SDK test framework](#using-the-azure-python-sdk-test-framework)
```

Comment:
not sure if this should be showing a link like the above sections, but it does not.